### ISSUE TITLE
Open selected path in application

### ISSUE THREAD

**Issue by FreeHarry** [fabiospampinato/vscode-open-in-application#16]
Please, support the possibility to open-in-application from a selected path in the editor.

**Comment by fabiospampinato** [comment]
What do you mean? Like right-clicking a path embedded in the editor?

A screenshot could be helpful.

**Comment by FreeHarry** [comment]
Yes, you have an embedded path in the editor as you can see on the picture...

<img width="683" alt="2022-02-16_10-06-14--Code" src="https://user-images.githubusercontent.com/24735031/154232277-c82d6d29-cd4e-4033-91c0-ed91d037d77e.png">

This path can be of any type (txt, html, ...). Via context menu entry you can open it. If there is a selection in the editor interpret it as the path and try to open it in default app. If there is no selection try to extract it from the current cursor location.

Path can be a file path or a directory. If it is a directory open it in the file manager.

Supporting of environment variables and relative paths (e.g. relative to the current editor or workspace) would be also great.
